index.html

```diff
@@ -1,45 +1,132 @@
 
-        </nav>
-    </header>
-</main>
-<!-- Read About My Life Section [SARA MAGDY]-->
-<section class="about section--py section"></section>
+        <!-- End about__img -->
 
-<!-- My Work Process Section [Mohamed Saeed Saafan]-->
-<section class="work-process section--py section"></section>
+        <div class="about__content">
+          <h2 class="about__title">
+            read about my life <br />
+            struggle story
+          </h2>
+          <p class="about__desc">
+            Donec commodo augue vitae metus interdum molestie. Fusce dapibus non
+            mauris sit amet blandit. Maecenas quis massa nec sem laoreet
+            vehicula quis sit amet tortor. Proin ornare, lorem ut pellentesque
+            maximus, ex leo viverra lorem, ac dapibus neque tellus id dolor.
+            Curabitur luctus tortor eget dolor ultrices fermentum.
+          </p>
+          <!-- progress div -->
+          <div class="about__progress">
+            <div class="about__item about__item--development">
+              <h3 class="about__subtitle">
+                <i class="fas fa-code about__icon about__icon--dev"></i
+                >Development
+                <span class="about__precent">80%</span>
+              </h3>
+              <div class="about__proBG">
+                <span
+                  class="about__techne about__tech--dev"
+                  style="width: 80%"
+                ></span>
+              </div>
+            </div>
+          </div>
+          <!-- End development -->
+          <div class="about__item about__item--ui">
+            <h3 class="about__subtitle">
+              <i class="fas fa-tv about__icon about__icon--ui"></i>UI/UX Design
+              <span class="about__precent">65%</span>
+            </h3>
+            <div class="about__proBG">
+              <span
+                class="about__techne about__tech--ui"
+                style="width: 65%"
+              ></span>
+            </div>
+          </div>
+          <!-- End ui -->
+          <div class="about__item about__item--digital">
+            <h3 class="about__subtitle">
+              <i class="fas fa-link about__icon about__icon--dig"></i>Digital
+              Marketing
+              <span class="about__precent">45%</span>
+            </h3>
+            <div class="about__proBG">
+              <span
+                class="about__techne about__tech--dig"
+                style="width: 45%"
+              ></span>
+            </div>
+          </div>
+          <!-- End digital -->
+          <div class="about__item about__item--graphic">
+            <h3 class="about__subtitle">
+              <i class="fas fa-radiation about__icon about__icon--graph"></i
+              >Graphic Design
+              <span class="about__precent">55%</span>
+            </h3>
+            <div class="about__proBG">
+              <span
+                class="about__techne about__tech--graph"
+                style="width: 55%"
```

Comment:
why do u use style in line ?

Reply:
To represent the percentage in span progress . if i remove it all precentage width will be 100%